### PR TITLE
[Security] Customize generated Login Link

### LINKED ISSUE #15147: [Security] LoginLink with specific locale
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#40153
| PR author(s) | @roromix
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by roromix**
Fix #15147 and add documentation for symfony/symfony#40641

### CODE REVIEW

File: security/login_link.rst [hunk 1]
```diff
@@ -654,3 +654,52 @@ user create this POST request (e.g. by clicking a button)::
             <button type="submit">Continue</button>
         </form>
     {% endblock %}
+
+Customize generated Login Link
+------------------------------
+
+.. versionadded:: 5.3
+
+    This feature was introduced in Symfony 5.3.
```
Comment (OskarStark) [inline security/login_link.rst]: The possibility to customize the login link was introduced in Symfony 5.3.

File: security/login_link.rst [hunk 2]
```diff
@@ -654,3 +654,52 @@ user create this POST request (e.g. by clicking a button)::
...
+
+                // create a login link for $user this returns an instance
+                // of LoginLinkDetails
+                $loginLinkDetails = $loginLinkHandler->createLoginLink($user, $request);
+                $loginLink = $loginLinkDetails->getUrl();
+
+                // ... send the link and return a response (see previous section)
+            }
+
+            // if it's not submitted, render the "login" form
```
Comment (OskarStark) [inline security/login_link.rst]: This comment can be removed

File: security/login_link.rst (hunk 1 above)
Comment (roromix) [reply security/login_link.rst]: Corrected

File: security/login_link.rst (hunk 2 above)
Comment (roromix) [reply security/login_link.rst]: Removed

**Comment by roromix** [comment]
While writing the documentation I noticed that the RequestContext :: fromRequest method does not add the '_locale' parameter. I will therefore make a new pull request to pass a RequestContext rather than a Request.

**Comment by javiereguiluz** [comment]
Romain, thanks a lot for contributing these docs ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**